bindings/pyroot/pythonizations/python/ROOT/JsMVA/JsMVAMagic.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, shell):
     @magic_arguments()
     @argument("arg", nargs="?", default="on", help="Enable/Disable JavaScript visualisation for TMVA")
     def jsmva(self, line):
-        from JsMVA.JPyInterface import functions
+        from ROOT.JsMVA.JPyInterface import functions
```

Comment:
This is fixed in master branch (using a relative import):
`from .JPyInterface import functions`